signer/src/error.rs

```diff
@@ -14,6 +13,10 @@ use crate::storage::model::SigHash;
 /// Top-level signer error
 #[derive(Debug, thiserror::Error)]
 pub enum Error {
+    /// Indicates that a sweep transaction with the specified txid could not be found.
+    #[error("sweep transaction not found: {0}")]
+    DecodeProtobuf(#[from] prost::DecodeError),
+
```

Comment:
Is this some leftover?

Reply:
Kinda, I had forgotten that I added it. Will update.